### ISSUE TITLE
biblio.json file not updated in two years

### ISSUE THREAD

**Issue by tabatkins** [WICG/admin#190]
Currently, [SpecRef](https://github.com/tobie/specref/blob/main/scripts/run-all) relies on this repo's `biblio.json` file for its WICG references. Is this still meant to be up-to-date? If so, did an automatic process fall over, or did the manual updater person let it fall off their plate?

Alternately, should SpecRef be referring to something different, maybe [WebRef](https://github.com/w3c/webref)? If so, we should retire this `biblio.json` file so nobody accidentally depends on it.

**Comment by marcoscaceres** [comment]
Yeah, I think it’s always been manually maintained (by random folks in the community).  I wonder if we should deprecate it, specially if we can get the same (or better) data from WebRef.

**Comment by tidoust** [comment]
Oh, I hadn't seen this issue. Specref **already** pulls data from [browser-specs](https://github.com/w3c/browser-specs/) (the list of specs used in Webref), see related [pull script](https://github.com/tobie/specref/blob/main/scripts/browser-specs.js). The script only adds WICG entries when they don't exist in WICG's `biblio.json` file. There are ~50 of them in practice (e.g., `DIGITAL-GOODS`), which tends to confirm that `biblio.json` is not up-to-date, or not meant to be comprehensive.

If the file isn't used for other purpose, it would be good to retire the file entirely. That would also simplify maintenance from a Specref perspective (the pull scripts don't handle well the case when a Specref entry that was contributed by browser-specs starts appearing in `biblio.json` afterwards).

I will make sure that all entries in `biblio.json` have a corresponding entry in browser-specs, and will prepare a PR against Specref to drop the related WICG pull script.

**Comment by tidoust** [comment]
Here is a short comparison.

First, the entries that are still missing in browser-specs:
- [AOM](https://wicg.github.io/aom/spec/): in monitor list
- [ISOLATED-CONTEXTS](https://wicg.github.io/isolated-web-apps/isolated-contexts): about to be added to browser-specs, see https://github.com/w3c/browser-specs/pull/1467
- [MULTICAPTURE](https://wicg.github.io/multicapture/): in monitor list

I'll add them to browser-specs.

Then, entries that are missing from browser-specs, but for which the underlying incubation was abandoned or moved somewhere else:
- [BUDGET-API](https://wicg.github.io/budget-api/): no longer pursued
- [CORS-RFC1918](https://wicg.github.io/cors-rfc1918/): now known as "Private Network Access"
- [FRAME-TIMING](https://wicg.github.io/frame-timing/): no longer pursued
- [PRIORITY-HINTS](https://wicg.github.io/priority-hints/): no longer pursued
- [VISUAL-VIEWPORT](https://wicg.github.io/visual-viewport/): merged into CSSOM View

Specref entries should be manuallly updated to note deprecation.

And then, one entry known in browser-specs under a different name:
- [WEB-BACKGROUND-SYNC](https://wicg.github.io/background-sync/spec/): known as `background-sync`

Specref's entry should be made an alias of the one in browser-specs.

Note: tests in Specref ensure that no entry ever gets removed. No risk of accidently removing an entry from Specref!

**Comment by tidoust** [comment]
Specref no longer pulls any data from `biblio.json`. And browser-specs now contributes all active WICG entries to Specref.
Anything else holding #198?

**Comment by tabatkins** [comment]
Nope, I think we're good to kill it.

**Comment by tidoust** [comment]
Closing as file has now been deleted.
